apps/ffdweb-site/src/app/_components/PageHeader.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from 'next/image'
+import type { StaticImageData } from 'next/image'
+
+import { Button } from '@filecoin-foundation/ui/Button'
+import { Heading } from '@filecoin-foundation/ui/Heading'
+import { buildImageSizeProp } from '@filecoin-foundation/utils/buildImageSizeProp'
+import type { CTAProps } from '@filecoin-foundation/utils/types/ctaType'
+import { clsx } from 'clsx'
+
+import { BASE_DOMAIN } from '@/constants/siteMetadata'
+
+type TitleProps = {
+  children: string
+  fontVariant?: PageHeaderFontVariant
+}
+
+export type StaticImageProps = {
+  data: StaticImageData
+  alt: string
+}
+
+type PageHeaderImageProps = StaticImageProps
+
+type PageHeaderFontVariant = 'default' | 'homepage'
+
+type PageHeaderProps = {
+  title: TitleProps['children']
+  image: PageHeaderImageProps
+  cta?: CTAProps
+  mobileLayout?: 'content-above-image' | 'image-above-content'
+  fontVariant?: PageHeaderFontVariant
```

Comment:
We can check the path instead of passing a prop. See CR.

Reply:
Apologies - We can't use `path` without making this a client component. 

We could do something like the following but don't think it's worth the effort at the moment. I'll revert. 🙃🙏🏼

```

PageHeader.Title = function Title({ children }: TitleProps) {
  return (
    <Heading tag="h1" variant="4xl">
      {children}
    </Heading>
  )
}

HomepagePageHeader.Title = function HomepageTitle({ children }: TitleProps) {
  return (
    <Heading tag="h1" variant="5xl-fluid">
      {children}
    </Heading>
  )
}
```

Reply:
Nit - using `isHomepage` as a prop now..